tools/perl-version-convert

```diff
@@ -81,6 +81,8 @@ function convert_version
 	[[ "$DIST" == "Mail-AuthenticationResults" && "${VERSION:0:2}" == "2." ]] && ((${#VERSION} == 12)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VER[4]=${VERSION:11:1} && VERSION=${VERSION:0:1}
 	[[ "$DIST" == "Mail-DKIM" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 10)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VERSION=${VERSION:0:1}
 	[[ "$DIST" == "Mail-DKIM" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 12)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VER[4]=${VERSION:11:1} && VERSION=${VERSION:0:1}
+	[[ "$DIST" == "Mail-DMARC" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 10)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VERSION=${VERSION:0:1}
```

Comment:
Why "1." only?  Versions "0." with the same length (10) use the same versioning scheme.

Reply:
I just took Mail::DKIM as an example which also some 0.x version. But we should not cover at all 0.x version because they are ancient (~10 years)

Reply:
Re Mail::DKIM and "0." versions: they are covered by default rules, so we do not need special handling for them here.

Reply:
The 0.x should create an error as other modules depending on Mail::DMARC may fail because of incompatiblity.

Reply:
> The 0.x should create an error as other modules depending on Mail::DMARC may fail because of incompatiblity.

Sorry, this makes no sense.  `perl-version-convert` is not a tool to enforce compatibility between modules.

Reply:
but still can help